### PR TITLE
fix: i18n config cache on vuex

### LINKED ISSUE #4727: service worker / pwa fetch new update doesn't show up
It seems after the update of 
- #4713 

it doesn't update the client anymore. 
I did not get any new toast, or at least I still have on https://beta.kodadot.xyz all languages 
yet when I open it in incognito, I see only a new one 

current state
image

incognito 
image

I guess it's because it doesn't generate proper sw.js ? service worker where old instance was listening? 
edit seems is there, old one doesn't switch for new one?
- https://beta.kodadot.xyz/sw.js

image

## Documentation

Probably needs custom workbox service worker? 

- https://github.com/kevinmarrec/nuxt-pwa-module

- https://github.com/kodadot/nft-gallery/blob/main/nuxt.config.js#L246 🤔

## Ref 
- #4723

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #4727
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: utils/config/i18n.ts [hunk 1]
```diff
@@ -2,10 +2,123 @@ import { LocaleMessages } from 'vue-i18n'
...
+  //   value: 'cz',
+  //   flag: '🇨🇿',
+  //   label: 'Česky',
+  // },
+  {
+    value: 'es',
+    flag: '🇪🇸',
+    label: 'Español',
+  },
+  {
```
Comment (vikiival) [inline utils/config/i18n.ts:36]: Remove commented?

File: utils/config/i18n.ts [hunk 2]
```diff
@@ -2,10 +2,123 @@ import { LocaleMessages } from 'vue-i18n'
...
+]
+
 const md = MarkdownIt({
   breaks: false,
 })
 
+export const getLangsFlags = () => langsFlags
+
+export const getUserFlag = (userLang: string) =>
+  langsFlags.find((lang) => lang.value === userLang)?.flag || langsFlags[0].flag
```
Comment (vikiival) [inline utils/config/i18n.ts:120]: Why also flag?

Comment (Jarsen136) [reply utils/config/i18n.ts]: updated

Comment (vikiival) [reply utils/config/i18n.ts]: ?

Comment (Jarsen136) [reply utils/config/i18n.ts]: I think `getUserFlag ` could stay in its original place.

**Comment by yangwao** [comment]
pay 50 usd 

let's try if it updates my instance on beta with new stuff

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 34.25 USD/KSM ~ 1.46 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0x3c6f96278ed155d86d0196f59d5e204b6066c72b12b425649f3ee8ee07f38875](https://kusama.subscan.io/extrinsic/0x3c6f96278ed155d86d0196f59d5e204b6066c72b12b425649f3ee8ee07f38875)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by yangwao** [comment]
pay 50

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 35.12 USD/KSM ~ 1.424 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0xe76f4575c5d23b9792ecc424b7349215c2997ce7944b9a43add3355a1c7a245d](https://kusama.subscan.io/extrinsic/0xe76f4575c5d23b9792ecc424b7349215c2997ce7944b9a43add3355a1c7a245d)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**